ontopy/excelparser.py

```diff
@@ -155,17 +152,14 @@ def create_ontology_from_pandas(  # pylint: disable=too-many-locals,too-many-bra
 
 def get_metadata_from_dataframe(  # pylint: disable=too-many-locals,too-many-branches,too-many-statements
     metadata: pd.DataFrame,
-    onto: owlready2.Ontology = None,
+    base_iri: str,
     base_iri_from_metadata: bool = True,
     catalog: dict = None,
 ) -> Tuple[ontopy.ontology.Ontology, dict]:
     """
-    Populate ontology with metada from pd.DataFrame
+    Create ontology with metadata from pd.DataFrame
     """
 
-    if onto is None:
```

Comment:
What happens now if base_iri is not given in the excelsheet? I think that the parser should not necessarily fail if it is not given

Reply:
Good point. I pushed a minor fix to the excelparser-update branch to also include IndexError in the except statement on line 172 to make sure that the ontology will get an default IRI rather than raising an exception.